### ISSUE TITLE
Esc key does not exit action in Ps CC2017 on OSX 10.13

### ISSUE THREAD

**Issue by frontenddevguy** [tashian/multiverse#1]
Thanks for making a great creative tool!

I've checked the esc key is working within Photoshop for other functions (and disabled as text commit).

To exit the script I have to force quit Ps.

Also, the rename layers script only works on one layer at a time. Is this expected behaviour?

**Comment by frontenddevguy** [comment]
Hm, esc still not working at all here with Sierra and CC2017.
cmd+s just saves and the script continues. 
shift+cmd+s does allow me to save the layer state with accurate timing :)
Only from the save as dialog does esc have the expected behaviour on my system.

The text file has lots of lines, double line breaks etc, but still only changes the currently selected layer.
I actually had a spam text file I saved from way back too.

So I guess Ps scripting is very environment specific..

Thanks

**Comment by tashian** [comment]
Yeah one thing I noticed is that the scripts ran the same across different versions of PS all the way back to 2015 (they really haven't changed the JS engine), _but_ it ran differently on different machines. May have just been different OS versions though—I never got to the bottom of it.
